src/config/Option.test.ts

```diff
@@ -0,0 +1,181 @@
+import { CodePointBuffer, Parameter } from "../token/Tokens";
+import { BooleanOption, EnumOption, RangeOption } from "./Option";
+
+test("BooleanOption Get", () => {
+  const option = new BooleanOption("test", true);
+  expect(option.key).toBe("test");
+  expect(option.defaultValue).toBe(true);
+  expect(option.getValue()).toBe(true);
+  expect(option.getUsageDescription()).toBe("TRUE | false");
+  const option2 = new BooleanOption("test", false);
+  expect(option2.defaultValue).toBe(false);
+  expect(option2.getValue()).toBe(false);
+  expect(option2.getUsageDescription()).toBe("true | FALSE");
+});
+
+test("BooleanOption Set Boolean", () => {
```

Comment:
change to `BooleanOption set & setter function`

Reply:
Updated, please refer to 
[c14362b](https://github.com/Jerrylum/protocol-diagram/pull/8/commits/c14362b46accd850aaa7de3765b4c67e42df8132)